index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio Site</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="//use.fontawesome.com/releases/v5.0.7/css/all.css">
+  
+</head>
+<body>
+  <header class="head">
+    <nav>
+      <h3 class="logo">Portfolio</h3>
+      <i class="fa fa-bars"></i>   
+    </nav>
+  </header>
+  <main>
+    <div class="container">
+      <h1>
+      I'm Uchechi<br>
+      Glad to see<br>
+      you!
+      </h1>
+      <p>
+      I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <div class="connect">
+        <h4>LETS CONNECT</h4>
+        <img src="images/Tweeter.png">
+        <img src="images/Linkedin icon.png">
+        <img src="images/m.png">
+        <img src="images/Github.png">
+        <img src="images/tech.png">
+      </div>
```

Comment:
- [ ] Great on implementing list items in a div, but it's better to use semantic unorder list `ul` for all the listed items inside the div for better code readability, user experience, and greater accessibility. Kindly use a  `ul` tag instead of a `div`. :+1:

Reply:
Done